<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRandomData2</class>
 <widget class="QDialog" name="DialogRandomData2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Group 1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QFormLayout" name="formLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="labelN1">
               <property name="text">
                <string>N1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSpinBox" name="spinBoxN1">
               <property name="minimum">
                <number>30</number>
               </property>
               <property name="value">
                <number>50</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>means</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDoubleSpinBox" name="doubleSpinBox1Mux"/>
             </item>
             <item row="2" column="0">
              <widget class="QDoubleSpinBox" name="doubleSpinBox1Muy"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Covariance</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDoubleSpinBox" name="doubleSpinBox1Covar00">
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox1Covar01"/>
             </item>
             <item row="2" column="0">
              <widget class="QDoubleSpinBox" name="doubleSpinBox1Covar10"/>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox1Covar11">
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Group 2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="labelN2">
               <property name="text">
                <string>N2</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSpinBox" name="spinBoxN2">
               <property name="minimum">
                <number>30</number>
               </property>
               <property name="value">
                <number>50</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>means</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDoubleSpinBox" name="doubleSpinBox2Mux">
               <property name="value">
                <double>3.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QDoubleSpinBox" name="doubleSpinBox2Muy">
               <property name="value">
                <double>6.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_4">
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Covariance</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDoubleSpinBox" name="doubleSpinBox2Covar00">
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox2Covar01"/>
             </item>
             <item row="2" column="0">
              <widget class="QDoubleSpinBox" name="doubleSpinBox2Covar10"/>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox2Covar11">
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBoxN1</tabstop>
  <tabstop>doubleSpinBox1Mux</tabstop>
  <tabstop>doubleSpinBox1Muy</tabstop>
  <tabstop>doubleSpinBox1Covar00</tabstop>
  <tabstop>doubleSpinBox1Covar01</tabstop>
  <tabstop>doubleSpinBox1Covar10</tabstop>
  <tabstop>doubleSpinBox1Covar11</tabstop>
  <tabstop>spinBoxN2</tabstop>
  <tabstop>doubleSpinBox2Mux</tabstop>
  <tabstop>doubleSpinBox2Muy</tabstop>
  <tabstop>doubleSpinBox2Covar00</tabstop>
  <tabstop>doubleSpinBox2Covar01</tabstop>
  <tabstop>doubleSpinBox2Covar10</tabstop>
  <tabstop>doubleSpinBox2Covar11</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogRandomData2</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogRandomData2</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
